.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] In this project, you are supposed to use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) means first you would make a **development** branch from your **main/master** branch and then you would make your **development** branch your default branch, and then you would make a **feature-branch** from your **development** branch and after making changes into that **feature-branch** you would ask for a review from the same **feature-branch** then after getting approval you would merge that **feature-branch** into the **development** branch :blush:

You can also check this [document](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md) which has been given to you on how to ask for a Code Review :+1:

### Note: If you make a new branch and a new pull request then you can reference this pull request in your new pull request summary to let the next reviewer know that your project has already been through a review so he/she will act accordingly :+1:

Reply:
Hi @HAMMAS-SALEEM I think this PR is not invalid. For all capstone projects since javascript, the PR opened from the `dev` branch to the `master/main` branch for the final external review. This was mentioned in the guidelines of the javascript capstone project. I did that since I was in the program for any capstone project request for a review in capstone and also my colleagues from Microverse did that. This is my PR for the same project. https://github.com/gemmen29/budget-app/pull/4

Reply:
Even when I checked your profile I saw you did the same in the same project https://github.com/HAMMAS-SALEEM/MyBudget/pull/13 and it was reviewed normally.